js/theme.js

```diff
@@ -117,7 +117,7 @@ function customParallaxImage(){
 }
 /* End Parallax Image */
 /* Start Smooth Scrolling */
-function customSmoothScrolling(){
+async function customSmoothScrolling(){
```

Comment:
This async is not the same as the async for loading scripts.  This doesn't guarantee the dependencies are available - I'm not even sure this call structure works.